### PR TITLE
Bot for collecting data using API

### PR DESCRIPTION
**Pull request by tavipoldma**
initial version for issue #123

### CODE REVIEW

**Comment by wagner-certat** [comment]
Any chance to get progress here? Especially *docs*.

cc @haam3r

**Comment by tavipoldma** [comment]
excuses for delay / winter sleep. start and stop issue is now resolved, please check
@wagner-certat

File: intelmq/bots/collectors/api/REQUIREMENTS.txt [hunk 1]
```diff
@@ -1 +1 @@
-tornado>=4.5.2
+tornado>=5.1
```
Comment (wagner-certat) [inline intelmq/bots/collectors/api/REQUIREMENTS.txt]: Is this because you only tested it with this version or do the changes require the new version?

Comment (tavipoldma) [reply intelmq/bots/collectors/api/REQUIREMENTS.txt]: both, changes are using the latest tornado syntax and only tested for that version

Comment (wagner-certat) [reply intelmq/bots/collectors/api/REQUIREMENTS.txt]: Interesting, I successfully tested it with 4.5.3

**Pull request merged**